src/actions.js

```diff
@@ -0,0 +1,32 @@
+export const inferResponseLimit = function(response) {
+  const count = response.count;
+  const resultCount = response.results.length;
+
+  if (count > resultCount) {
+    return resultCount;
+  }
+
+  return null;
+};
+
+export const parseResponse = function(response) {
```

Comment:
I'm curious why this is called `parseResponse` when it seems to just be adding page count and limit attributes to the response?